cloudtrail/lib/botocore/data/s3/2006-03-01/examples-1.json

```diff
@@ -0,0 +1,1843 @@
+          ]
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following example return versions of an object with specific key name prefix. The request limits the number of items returned to two. If there are are more than two object version, S3 returns NextToken in the response. You can specify this token value in your next request to fetch next set of object versions.",
+        "id": "to-list-object-versions-1481910996058",
+        "title": "To list object versions"
+      }
+    ],
+    "ListObjects": [
+      {
+        "input": {
+          "Bucket": "examplebucket",
+          "MaxKeys": "2"
+        },
+        "output": {
+          "Contents": [
+            {
+              "ETag": "\"70ee1738b6b21e2c8a43f3a5ab0eee71\"",
+              "Key": "example1.jpg",
+              "LastModified": "2014-11-21T19:40:05.000Z",
+              "Owner": {
+                "DisplayName": "myname",
+                "ID": "12345example25102679df27bb0ae12b3f85be6f290b936c4393484be31bebcc"
+              },
+              "Size": 11,
+              "StorageClass": "STANDARD"
+            },
+            {
+              "ETag": "\"9c8af9a76df052144598c115ef33e511\"",
+              "Key": "example2.jpg",
+              "LastModified": "2013-11-15T01:10:49.000Z",
+              "Owner": {
+                "DisplayName": "myname",
+                "ID": "12345example25102679df27bb0ae12b3f85be6f290b936c4393484be31bebcc"
+              },
+              "Size": 713193,
+              "StorageClass": "STANDARD"
+            }
+          ],
+          "NextMarker": "eyJNYXJrZXIiOiBudWxsLCAiYm90b190cnVuY2F0ZV9hbW91bnQiOiAyfQ=="
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "The following example list two objects in a bucket.",
+        "id": "to-list-objects-in-a-bucket-1473447646507",
+        "title": "To list objects in a bucket"
+      }
+    ],
+    "ListObjectsV2": [
+      {
+        "input": {
+          "Bucket": "examplebucket",
+          "MaxKeys": "2"
+        },
+        "output": {
+          "Contents": [
+            {
+              "ETag": "\"70ee1738b6b21e2c8a43f3a5ab0eee71\"",
+              "Key": "happyface.jpg",
+              "LastModified": "2014-11-21T19:40:05.000Z",
+              "Size": 11,
+              "StorageClass": "STANDARD"
+            },
+            {
+              "ETag": "\"becf17f89c30367a9a44495d62ed521a-1\"",
+              "Key": "test.jpg",
+              "LastModified": "2014-05-02T04:51:50.000Z",
+              "Size": 4192256,
+              "StorageClass": "STANDARD"
+            }
+          ],
+          "IsTruncated": true,
+          "KeyCount": "2",
+          "MaxKeys": "2",
+          "Name": "examplebucket",
+          "NextContinuationToken": "1w41l63U0xa8q7smH50vCxyTQqdxo69O3EmK28Bi5PcROI4wI/EyIJg==",
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 329716fe-64c0-4be7-ba35-37fa1db94d57 -->